modules/virt-creating-vm-snapshot-cli.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc
+
+[id="virt-creating-vm-snapshot-cli_{context}"]
+= Creating an virtual machine snapshot in the CLI
+
+You can create a virtual machine (VM) snapshot for an offline or online VM by creating a `VirtualMachineSnapshot` object. Kubevirt will coordinate with the QEMU guest agent to create a snapshot of the online VM.
+
+[NOTE]
+====
+In order to create snapshots of an online (Running state) VM with the highest integrity, we recommend you install the QEMU guest agent.
```

Comment:
Good.  Can we copy this to the other places?

Reply:
Yep, it's in all the renamed files.